Avoid errors during runtime if devel/py-packaging is not present.

The package is only required in "_version_cls.py" and falls back to
devel/py-setuptools (via "try ... except") if it's not available.

This is a workaround to remedy issues with upgrade runs that also update
devel/py-{packaging,pyparsing} in environments where devel/py-setuptools_scm is
already present.

--- setup.cfg.orig	2021-12-05 20:43:43 UTC
+++ setup.cfg
@@ -27,7 +27,6 @@ classifiers = 
 [options]
 packages = find:
 install_requires = 
-	packaging>=20.0
 	setuptools
 	tomli>=1.0.0 # keep in sync
 python_requires = >=3.6
